README.md

```diff
@@ -0,0 +1,189 @@
+
+
+### Key Features <a name="key-features"></a>
+
+- **Linters SetUp**
+- **Follow HTMl and javascript best practice**
+- **Creationg component**
+- **Follow Git flow**
+- **initialising APP structure**
+- **Fetching data from API**
+
+## 🛠 Built With <a name="built-with"></a>
+   React
+   CSS
+   ES6
+   JavaScript
+   Redux
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+[Live Demo](https://wol-snn6.onrender.com)
+
+
+<!--Video-->
+## Video Presentation <a name="video-presentation"></a>
+[Click Here for Video Presentation])(https://www.loom.com/share/093d856ac7484ab6bff87bfe147bd992)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  https://github.com/wolterbwambale/metrics-webapp-capstone
+  cd ForeignCurrency-app
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  npm install
+```
+
+### Usage
+
+To run the project in a development server, execute the following command:
+
+```sh
+  npm start
+```
+
+### Testing
+
+To run the tests in a development server, execute the following command:
+
+```sh
+  npm test
+```
+
+To to build for the production, execute the following command:
+
+```sh
+  npm run build
+```
+
+### Deployment
+
+You can deploy this project using [Github Pages](https://docs.github.com/en/pages/getting-started-with-github-pages/creating-a-github-pages-site)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+👤 **Bwambale**
+  
+- GitHub: [Bwambale](https://github.com/wolterbwambale/metrics-webapp-capstone)
+- LinkedIn: [Bwambale](https://www.linkedin.com/in/bwambale-benny-wolter-a9284925a/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Mobile version**
+- [ ] **Desktop version**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/wolterbwambale/metrics-webapp-capstone/issues).
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+
+## ⭐️ Show your support <a name="support"></a>
+
+Give a ⭐️ if you like this project! thankyou 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> I would like to thank the Microverse community for their support.
+
+ I would like to thank Nelson Sakwa . the author of the original design
```

Comment:
- [ OPTIONAL] Good work giving credit where its due. I would suggest that you make **Nelson Sakwa** & **original design** links to te respective entities 😄